### PR TITLE
gitignore: add cmake-build-debug folder

### PR DESCRIPTION
**Pull request by timeyyy**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Build/deps dir
 /build/
+/cmake-build-debug/
```
Comment (justinmk) [inline .gitignore]: where'd this come from?

Comment (timeyyy) [reply .gitignore]: I'm not sure.. i'll just delete the folder for now i guess

Comment (fwalch) [reply .gitignore]: CLion creates build folders with this name, I think.

Comment (justinmk) [reply .gitignore]: oh, we can ignore if it's coming from a build tool and not a personal config.

**Comment by timeyyy** [comment]
can confirm it is from clion

**Pull request merged**